### DISCUSSION TITLE
Allow monthly sponsors with no commitment

### DISCUSSION THREAD

**Discussion by SVillette** [community/community#59761] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi there,

I have a yearly billing plan (GH pro) and I wanted to sponsor someone for $5/month but the payment amount is calculated with prorated amount of the billing cycling

![image](https://github.com/community/community/assets/17042730/52d5c6cb-c501-4b56-9537-7212243376c3)

I understand my GH pro plan is billed yearly but I won't sponsor someone monthly to be billed yearly. I wish the sponsor system will change for that case.

**Comment by anjuan** [comment]
Hi, @SVillette, I'm the Engineering Manager for GitHub Sponsors. I understand how it can be difficult to want to sponsor now and have to wait for the full amount to go through. However, I don't see us making a change any time soon due to our current priorities. If we do any work in how we work within billing cycles, I'll keep this in mind.

**Reply by SVillette** [reply]
I think you misunderstand what I meant. I don't want to wait for the full (yearly) amount debited. I want to pay every month, not once a year (even with a yearly GH pro subscription)

**Reply by anjuan** [reply]
You can do a recurring sponsorship (assuming the maintainer allows monthly sponsorships).

**Reply by SVillette** [reply]
That's actually the issue. I chose montly sponsor but get the amount debited each year.
And I won't sponsor him because of this behaviour.

**Reply by anjuan** [reply]
Thanks for the feedback. I'll take this back to the team.

**Reply by beyera** [reply]
@anjuan: I have the same issue. I'm trying to sponsor a project at $500 per month, but I'm being charged annually. It seems excessive to commit to a year without knowing where the project will be. I tried switching our account to monthly billing, but I have to wait until the end of the annual term for that to take effect, including for sponsorships.

<img width="978" alt="image" src="https://github.com/user-attachments/assets/9cb7c5be-1433-4d1c-ab09-5853bd5ceab0">

**Comment by xames3** [comment]
So @anjuan is the prorated billing calculated for the billing cycle of the sponsor (payee) or the account owner (payable)? Sorry, but I'm still confused with the talk.

**Reply by anjuan** [reply]
It's based on the billing cycle of the sponsor.
